mesonpy/__init__.py

```diff
@@ -758,6 +758,11 @@ def __init__(
             if not value:
                 self._limited_api = False
 
+            if self._limited_api:
+                if bool(sysconfig.get_config_var("Py_GIL_DISABLED")):
```

Comment:
We use single quotes for strings. This is why pre-commit-hook is complaining.

Reply:
Yeah ... always takes me one iteration, since I am not touching pre-commit locally. Fixed now.